src/swarm/README_client_neo.rst

```diff
@@ -168,13 +168,13 @@ delegate looks something like this imaginary example:
     with ( info.Active ) switch ( info.active ) 
     {
       case success: // example success notification
-        Stdout.formatln("Request on channel {} succeeded and returned the "
+        Stdout.formatln("Request on channel {} succeeded and returned the " ~
             "value {}", args.channel, info.value);
```

Comment:
Sneaky fix in the same commit ;-).

Reply:
Couldn't pass by this outrageous recommendation in docs :)